pkg/dao/repository_configs.go

```diff
@@ -587,7 +606,7 @@ func (r repositoryConfigDaoImpl) bulkDelete(tx *gorm.DB, orgID string, uuids []s
 		var err error
 		var repoConfig models.RepositoryConfiguration
 
-		if repoConfig, err = r.fetchRepoConfig(orgID, uuids[i]); err != nil {
+		if repoConfig, err = r.fetchRepoConfig(orgID, uuids[i], false); err != nil {
```

Comment:
If I try to bulk delete a red hat repo, I correctly get the "not found" error. BUT it's returning a 500 response code instead of a 404

Reply:
I was able to fix this issue, I think we largely just weren't hitting it before, and it was pre-existing.